musl <stdlib.h> does not include <sys/types.h>, patch submitted upstream

fix a format string in ctl.c on 32-bit archs where size_t is unsigned int instead of long
--- a/tools/probes/probes_demux.c
+++ b/tools/probes/probes_demux.c
@@ -15,6 +15,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <unistd.h>
+#include <sys/types.h>
 
 #include <ipc/probe_dma_frame.h>
 
--- a/tools/ctl/ctl.c
+++ b/tools/ctl/ctl.c
@@ -503,7 +503,7 @@ static int ctl_set_get(struct ctl_data *ctl_data)
 		ref_size = ctl_data->buffer[BUFFER_SIZE_OFFSET] + BUFFER_TLV_HEADER_BYTES;
 		if (read_size != ref_size) {
 			fprintf(stderr,
-				"Error: Blob TLV header size %u (plus %lu) does not match with read bytes count %zu.\n",
+				"Error: Blob TLV header size %u (plus %zu) does not match with read bytes count %zu.\n",
 				ctl_data->buffer[BUFFER_SIZE_OFFSET], BUFFER_TLV_HEADER_BYTES,
 				read_size);
 			return -EINVAL;
